i18n/en/docusaurus-plugin-content-docs/current/identidad-digital.md

```diff
@@ -85,6 +87,8 @@ Integrating digital identity to blockchain networks opens new opportunities, for
 
 - Removing the complex private key mechanics from the user experience.
 
+You can learn about the [EOSIO DID method(decentralized identifier)](https://www.gimly.io/blog/the-eosio-did-method-specification), one more way to implement digital identity.
```

Comment:
### Digital identity on EOSIO 

There is an active working group developing an [EOSIO DID method (decentralized identifier)](https://www.gimly.io/blog/the-eosio-did-method-specification) which mat be a useful reference of existing EOSIO based implementations of digital identity.

Reply:
@AngeloCG97  lets add the H3 tag and replace text for the above